cypress/e2e/basket.spec.cy.js

```diff
@@ -0,0 +1,35 @@
+import BaseToolbarModule from '../pageObjects/modules/base.toolbar.module'
+import ProductsPage from '../pageObjects/products.page'
+import BasketPage from '../pageObjects/basket.page'
+
+describe('Basket tests', function () {
+  beforeEach(function () {
+    cy.loginViaApi(Cypress.env('username'), Cypress.env('password'))
+    cy.visit('/#/search')
+    cy.closeWelcomeBanner()
+    cy.fixture('productData').then(productData => {
+      this.productData = productData
+    })
+  })
+
+  it('a logged in user should be able to add an item to basket', function () {
+    const toolbar = new BaseToolbarModule()
```

Comment:
I think you can create objects in the describe and reuse them. At least I have not find any information it should affect the parallelization